docs/source/articles/extend-test-case-mixins.html.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Extend Test Case Mixins
+created_at: 2020/01/23
+excerpt: How to change existing test case mixins and add your own test case mixins
+---
+
+# Extend Test Case Mixins
+
+Workarea provides several [test case mixins](/articles/testing-concepts.html#test-case-mixins), which you can extend.
+
+This document provides procedures for the following:
+
+* [Change an Existing Test Case Mixin](#change-an-existing-test-case-mixin)
+* [Add a New Test Case Mixin](#add-a-new-test-case-mixin)
+
+
+## Change an Existing Test Case Mixin
+
+You may want to change a method within an existing test case mixin so that the change is reflected in all tests (rather than decorating specific tests).
+
+For example, say you want to change the email address filled in by [`Storefront::SystemTest#fill_in_email`](https://github.com/workarea-commerce/workarea/blob/v3.5.3/testing/lib/workarea/storefront/system_test.rb#L90-L92) from `bcrouse-new-account@workarea.com` to `robert-clams@workarea.com`.
+
+Create a new file in `/test/support/` and re-open the previously defined module and method, applying the email address change:
```

Comment:
I think its worth mentioning that any files in test/support will be loaded when the tests are run.